extensions/amp-video/0.1/amp-video.js

```diff
@@ -71,9 +75,12 @@ class AmpVideo extends AMP.BaseElement {
     /** @private {?Element} */
     this.video_ = null;
 
-    /** @private {?boolean}  */
+    /** @private {boolean}  */
```

Comment:
I think this could just be `/** @private */`. The type is unnecessary here because Closure Compiler infers the type when an initial primitive value or a typed function return value is assigned.

Reply:
true, most places seem to be explicit on the type though + consistency for the whole block